crux-llvm/test-data/golden/T972-fail.at-least-clang12.good

```diff
@@ -0,0 +1,6 @@
+[Crux] Found counterexample for verification goal
+[Crux]   test-data/golden/T972-fail.at-least-clang12.c:8:3: error: in main
+[Crux]   [callFunction] Unsupported function type: PtrTo (FunTy (PrimType Void) [PrimType (Integer 32),PtrTo (PrimType (Integer 8)),PtrTo (PrimType (Integer 8))] False)
```

Comment:
I'm a bit surprised that it fails with this error message, since this function type doesn't look that strange. (I'd expect it to eventually fail on the `ValAsm` bit, but it doesn't reach that part.) Perhaps the `CallBr` case in `generateInstr` should match on `PtrTo` like the `Call` case does?

The `Invoke` case should do this as well, now that I look closer.

Reply:
Good call, fixed!

Reply:
Huh, that didn't work for `invoke`:
```
        Exception: user error (Failure encountered while generating a Crucible CFG: at internal: unexpected function type in invoke: FunTy (PtrTo (Alias (Ident "class.std::__cxx11::basic_string"))) [PtrTo (Alias (Ident "class.std::__cxx11::basic_string")),PrimType (Integer 64),PrimType (Integer 64),PtrTo (PrimType (Integer 8)),PrimType (Integer 64)] False)
```
Nonetheless, I've fixed it for `callbr`.

Reply:
Wow, `invoke` gets more mysterious every time I look at it. But in any case, I'm much happier with the way the error message for the `callbr` test case looks now.

Reply:
Thinking about this some more, I think it's a bug that `invoke`'s function type isn't a `PtrTo` here. I've opened GaloisInc/llvm-pretty-bc-parser#189 for this.